tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/resource/N4JSResourceLoadStatesTest.xtend

```diff
@@ -208,7 +208,7 @@ class N4JSResourceLoadStatesTest extends AbstractN4JSTest {
 	}
 
 	@Test
-	def void testStateFullyProcessed3() throws Exception {
+	def void testStateFullyProcessed_triggeredOnlyThroughProxyResolution() throws Exception {
```

Comment:
FYI: I renamed this test because I now refer to it from the design document and therefore wanted to have a more meaningful and stable name than just "testStateFullyProcessed3".